File: 2026/Aspire13/code/02-js-ts-parity/DemoTS/apphost.ts
// Aspire TypeScript AppHost
// For more information, see: https://aspire.dev

import { createBuilder } from './.modules/aspire.js';

const builder = await createBuilder();

// Add your resources here, for example:
// const redis = await builder.addContainer("cache", "redis:latest");
// const postgres = await builder.addPostgres("db");
builder.addJavaScriptApp("myWebApp", "./web");
await builder.build().run();

File: 2026/Aspire13/code/05-telemetry-operations/apphost.cs
#:sdk Aspire.AppHost.Sdk@13.3.0
#:package Aspire.Hosting.Redis@13.3.0

var builder = DistributedApplication.CreateBuilder(args);
builder.AddRedis("cache");
#pragma warning disable ASPIRECSHARPAPPS001
builder.AddCSharpApp("worker", "./netcoreTel.cs");
#pragma warning restore ASPIRECSHARPAPPS001

builder.Build().Run();
